common/djangoapps/student/tests/test_authz.py

```diff
@@ -182,7 +182,7 @@ def setUp(self):
         self.global_admin = AdminFactory()
         self.creator = User.objects.create_user('testcreator', 'testcreator+courses@edx.org', 'foo')
         self.staff = User.objects.create_user('teststaff', 'teststaff+courses@edx.org', 'foo')
-        self.course_key = SlashSeparatedCourseKey('mitX', '101', 'test')
+        self.course_key = CourseLocator('mitX', '101', 'test')
```

Comment:
Is this okay? I thought we created old style keys specifically to test that we don't break old mongo compatibility.

I mean, I'm desperately for killing old mongo entirely at some point, but until then I thought we were just going to remove the `SlashSeparatedCourseKey.from_deprecated_string()` calls because people are copy-pasting that instead of using the backwards compatible `CourseKey.from_string()`.